README.md

```diff
@@ -4,6 +4,7 @@ Randomly spawning powerups
 ## Requirements
 
  - [CFC Logger](https://github.com/CFC-Servers/cfc_logger)
+ - [CFC ULX Commands](https://github.com/CFC-Servers/cfc_ulx_commands) for the Curse powerup
```

Comment:
I would much prefer this be an optional plugin

Reply:
It is optional, not having it just disables the Curse powerup. I'll make the note more clear though,